src/BusinessService.ts

```diff
@@ -834,6 +834,21 @@ class BusinessService extends Base {
     return json as unknown as BusinessHierarchyInvitation[];
   }
 
+  /**
+   * List all 'hierarchy' invitations by LBID id.
+   *
+   * @remarks should be used by child LBID admins.
+   */
+  async getOneChildrenInvitationByToken(token: BusinessHierarchyInvitation["token"]) {
+    const { apiEndpointAddress, fetch } = this.lensPlatformClient;
+    const url = `${apiEndpointAddress}/businesses/hierarchies/invitations?token=${token}`;
+    const json = await throwExpected(async () => fetch.get(url), {
+      404: (error) => new NotFoundException(error?.body?.message),
```

Comment:
Is it really only 404 possible ?

Reply:
Yes, this endpoint doesn't need authentication, so even 401/403 not possible.

Reply:
Thanks, 403 added.